tests/drivers/can/timing/src/main.c

```diff
@@ -193,6 +193,17 @@ static void test_timing_values(const struct device *dev, const struct can_timing
 		assert_timing_within_bounds(&timing, min, max);
 		assert_sp_within_margin(&timing, test->sp, SAMPLE_POINT_MARGIN);
 
+		if (IS_ENABLED(CONFIG_CAN_FD_MODE)) {
+			if (data_phase) {
+				err = can_set_timing(dev, can_get_timing_min(dev), &timing);
+			} else {
+				err = can_set_timing(dev, &timing, can_get_timing_min_data(dev));
+			}
+		} else {
+			err = can_set_timing(dev, &timing, NULL);
```

Comment:
Do you want to set the min timing here too?

Reply:
the test case is supposed to try to set `timing`, not `min_timing`. 
I guess `min_timing` is used in the can-fd case to avoid setting the same timing configuration for both data and regular timings.

Reply:
@karstenkoenig Exactly. We cannot set the same timing for both the arbitration and data phase as they may have different limits. Therefore, the data phase timing test simply uses the minimum timing allowed by the driver for the arbitration phase (at this point, that was already validated by another test case).